src/Database/Attach/Resizer.php

```diff
@@ -687,4 +687,14 @@ protected function getSizeByFit($maxWidth, $maxHeight)
 
         return [$optimalWidth, $optimalHeight];
     }
+
+    /**
+     * Return the filename extension for the provided filename
+     * @param string $path The filename path
+     * @return string
```

Comment:
We don't want to keep docblock typehints around if we have method signature typehints, makes it harder to maintain.